spec/predicates/template/template.md

```diff
@@ -0,0 +1,87 @@
+# Predicate type: <name>
+
+Type URI:
+
+Version:
+
+Predicate Name:
+
+## Purpose
+
+This section must discuss at a high level the reason for creating the new
+predicate type. Essentially, it describes the problems that can be solved using
+the new predicate.
+
+## Use Cases
+
+This section must discuss at least one concrete use case that will be covered by
+this predicate and how the current predicate types (if any) do not solve this.
+
+## Prerequisites
+
+New predicate types may be very specific to a particular technology or
+specification, which must be detailed in the prerequisites section. The in-toto
+Attestation Framework is likely a necessary prerequisite for all predicate
```

Comment:
I'm not sure if this is true anymore.

optional: "The in-toto Attestation Framework may be a necessary prerequisite... if so it SHOULD be explicitly..."

If you don't want to mess with it though that's fine.